iontestdata/good/equivs/structWhitespace.ion

```diff
@@ -0,0 +1,286 @@
+(
```

Comment:
What's the benefit/significance of using an s-expression as the container vs a list?

Reply:
Personal preference. S-expression elements can be delimited by whitespace, not commas, so there's less noise.